gematria/datasets/python/import_from_bhive.py

```diff
@@ -77,13 +77,19 @@
     'The index of the throughput value column in the input CSV file.',
 )
 
-flags.register_multi_flags_validator(
+
+@flags.multi_flags_validator(
     ['machine_code_hex_column_index', 'throughput_column_index'],
```

Comment:
There's a way to avoid using string literals for flag names: instead of `'machine_code_hex_column_index'`, it's better to use the flag holder to get the name with `_MACHINE_CODE_HEX_COLUMN_INDEX.name`.

Similar for other flag names.